src/import-map/import-map.component.js

```diff
@@ -25,9 +24,12 @@ export default function ImportMap(props) {
         </section>
         {editingImportMap
           ?
-            <EditImportMap
-              finished={finishedEditing}
-            />
+            <>
+              <div>
+                Click on the buttons below to view and modify your import map
+              </div>
+              <import-map-overrides-list></import-map-overrides-list>
```

Comment:
See [docs for import-map-overrides-list custom element](https://github.com/joeldenning/import-map-overrides#usage). There's no need for single-spa-playground to implement what's already implemented by import-map-overrides.